### PR TITLE
Add multirule support

### PR DESCRIPTION
**Pull request by IreneMLC**
### Relevant information

- Added support for multi-rules on the schema validation.
- Added multiple rules for INFRA-CONTAINER (and updated tags to our current ones).

### CODE REVIEW

File: definitions/infra-container/definition.yml
Comment (srvaroa) [inline definitions/infra-container/definition.yml:43]: Should we add the alias support before merging?

Comment (douglascamata) [reply definitions/infra-container/definition.yml:43]: +1

Comment (IreneMLC) [reply definitions/infra-container/definition.yml:43]: Sure! I was thinking of doing that next when I saw we needed this here (but the multi-rule changes took priority since they blocked the release of the V2 that @naxhh is working on). I'll wait to merge this until we have the aliases.

File: README.md
Comment (srvaroa) [inline README.md:65]: This sentence reads a bit confusing.  I would just describe what the synthesis section does and maybe just mention that some types (e.g. all INFRA- or APP-) do not allow them for historical reasons.

File: README.md
Comment (srvaroa) [inline README.md:67]: "You can define synthesis rules when your telemetry has consistent attributes that can uniquely identify an entity" seems clearer?

File: README.md
Comment (srvaroa) [inline README.md:76]: Unique within an account.

File: README.md
Comment (srvaroa) [inline README.md:77]: I would maybe put this one first

File: README.md
Comment (srvaroa) [inline README.md:78]: Indicate that we only support 1?

File: README.md
Comment (srvaroa) [inline README.md:83]: Reverse sentence? "When the value of the identifier attribute may be longer than X chars, you should set this to true. "

File: README.md
Comment (srvaroa) [inline README.md:85]: "Telemetry attributes that should be extracted into entity tags" ?

File: README.md
Comment (srvaroa) [inline README.md:102]: I would omit the comments for all the properties that were already described above, it'll make the sample easier to read

**Review by srvaroa — COMMENTED** [review]
Some suggestions for wording but LGTM!

**Review by naxhh — APPROVED** [review]

File: README.md
Comment (IreneMLC) [reply README.md:67]: 👍  This is just for the simplified format, I'll reword it.

File: README.md
Comment (IreneMLC) [reply README.md:102]: The two examples (simple and multi-rule) are collapsed by default, so you can go directly for the multi-rule one and the comments might help I think.

**Review by cianBuckley — APPROVED** [review]

**Pull request merged**
